### DISCUSSION TITLE
Content Definition versioning

### DISCUSSION THREAD

**Discussion by dafergu2** [OrchardCMS/OrchardCore#8760] [Ideas]
While chatting with @Skrypt about content definitions, I asked about the concept of content definition versioning.  My client has a desire to be able to rollback almost any mistake made when managing content.  Part of this is likely going to be centered around content definitions as there are quite a few new types and parts that we are creating to support the ease of use they desire when editing content and moving things around.  As such, I have an expectation of content parts and types being added and edited fairly often.

Would adding versioning to content definitions be something that the team as a whole would find useful?

**Comment by agriffard** [comment]
#1030
